### PR TITLE
first pass edits on install doc

### PR DESCRIPTION
**Pull request by rosemarymarano**
Please check content in the Load Balancing section (I copied the first paragraph from the README file). Particularly review this part in the section, "Note that this release provides only basic load balancing ...,"

### CODE REVIEW

File: site/installation.md [hunk 1]
```diff
@@ -64,64 +64,64 @@ Verify that you have the right image by running `docker images | grep webloogic-
...
-### Load Balancing
+### Load balancing with an Ingress controller or a web server
 
-The operator can install the Traefik Ingress provider to provide load balancing for web applications running in WebLogic clusters.  If enabled, an instance of Traefik and an Ingress will be created for each WebLogic cluster.  Additional configuration is performed when creating the domain.
+You can choose a load balancer provider for your WebLogic domains running in a Kubernetes cluster. Please refer to Load balancing with Voyager Ingress Controller, [Load balancing with Traefik Ingress Controller](site/traefik.md), and [Load balancing with Apache Web Server](site/apache.md) for information about the current capabilities and setup instructions for each of the supported load balancers.
 
-Note that the technology preview release provides only basic load balancing:
+Note that this release provides only basic load balancing:
 
 *	Only HTTP(S) is supported. Other protocols are not supported.
```
Comment (mriccell) [inline site/installation.md]: I think we need to identify which other protocols are not supported, T3, IIOP? I think now that we support HAPROXY and Apache we can remove the line
"Note that this release provides only basic load balancing:"
We need Shean to check this statement.

Comment (rosemarymarano) [reply site/installation.md]: Ok, I'll remove this line, ""Note that this release provides only basic load balancing:"

How do we get "Shean to check this statement."

Comment (mriccell) [reply site/installation.md]: I tried adding him for review but was not able.  I will have a meeting and discuss these 2 statements and then confirm with you if we should remove.

File: site/installation.md [hunk 2]
```diff
@@ -64,64 +64,64 @@ Verify that you have the right image by running `docker images | grep webloogic-
...
-| targetNamespaces | A list of the Kubernetes namespaces that may contain WebLogic domains that the operator will manage.  The operator will not take any action against a domain that is in a namespace not listed here. | default |
-| weblogicOperatorImage | The Docker image containing the operator code. | container-registry.oracle.com/middleware/weblogic-kubernetes-operator:latest |
-| weblogicOperatorImagePullPolicy | The image pull policy for the operator docker image.  Allowed values are 'Always', 'Never' and 'IfNotPresent' | IfNotPresent |
-| weblogicOperatorImagePullSecretName | Name of the Kubernetes secret to access the Docker Store to pull the WebLogic Server Docker image.  The presence of the secret will be validated when this parameter is enabled. | |
+| `elkIntegrationEnabled` | Determines whether the Elastic Stack integration will be enabled.  If set to `true`, then Elasticsearch, Logstash, and Kibana will be installed, and Logstash will be configured to export the operator’s logs to Elasticsearch. | `false` |
+| `externalDebugHttpPort` | The port number of the operator's debugging port outside of the Kubernetes cluster. | `3099`9 |
```
Comment (mriccell) [inline site/installation.md]: Typo `3099`9 fix to `30999`

Comment (rosemarymarano) [reply site/installation.md]: Done.

**Pull request merged**